src/documentation/0015-node-export-module/index.md

```diff
@@ -72,12 +72,6 @@ const items = require('./items');
 const { car } = items;
 ```
 
-or
-
-```js
-const { car } = require('./items');
```

Comment:
Think it can probably be dropped, but it looks like it was a problem when I added the ESLint to the samples. It was previously `const car = require('./items').car` which probably got autofixed to the destructoring like the next block